data_diff/diff_tables.py

```diff
@@ -381,6 +381,12 @@ def _bisect_and_diff_tables(self, table1, table2, level=0, max_rows=None):
         if max_rows < self.bisection_threshold:
             rows1, rows2 = self._threaded_call("get_values", [table1, table2])
             diff = list(diff_sets(rows1, rows2))
+
+            # This happens when the initial bisection threshold is larger than
+            # the table itself.
+            if level == 0 and not self.stats.get("table_count", False):
+                self.stats["table_count"] = self.stats.get("table_count", 0) + max(len(rows1), len(rows2))
```

Comment:
This fixes https://github.com/datafold/data-diff/issues/68

When the count is lower than the bisection threshold on the initial segment, the table count is never set.